package.json

```diff
@@ -29,5 +29,10 @@
   },
   "workspaces": [
     "apps/*"
-  ]
+  ],
+  "dependencies": {
+    "@nestjs/swagger": "^7.3.1",
+    "class-transformer": "^0.5.1",
+    "class-validator": "^0.14.1"
+  }
```

Comment:
these dependencies won't be used by the frontend app, so they should be in the package.json inside the apps/backend dir. Delete them from here, and execute the yarn add ... command from the apps/backend dir